### DISCUSSION TITLE
`fd` equivalent for finding multiple owners?

### DISCUSSION THREAD

**Discussion by pbe9394** [sharkdp/fd#1629] [Q&A]
Hello,

Is there a `fd` equivalent for something like `find ./ -user user1 -o -user user2 -o -user user3 ...` 

Thanks!

**Comment by ShreeChaturvedi** [comment]
fd's `--owner` only takes one user, and repeating the flag just keeps the last one. The simplest workaround is combining calls:

```bash
{ fd --owner user1; fd --owner user2; fd --owner user3; } | sort -u
```

`sort -u` deduplicates, though with distinct users you won't get overlap.

If you want a single pass instead, you can filter with `--exec`:

```bash
fd --exec sh -c 'case "$(stat -c %U "$1")" in user1|user2|user3) echo "$1";; esac' _ {}
```

That's slower because it shells out to `stat` on every match, so the multi-call version is usually better for large trees.
